SPECS/grub2/31_systemd

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+cat << EOF
+if [ -f  $GRUB_MARINER_BOOTPREFIX/systemd.cfg ]; then
```

Comment:
we should not use the systemd.cfg file anymore

Reply:
Do we not want systemd config changes to be configurable between systemd releases?

Reply:
the systemd.cfg file is the wrong way to do this; the correct way for non-grub packages to effect grub config is to place config files into /etc/default/grub.d/, so that *newly generated* grub.cfg will contain the changes